packages/build/src/homebrew.ts

```diff
@@ -1,78 +0,0 @@
-import crypto from 'crypto';
```

Comment:
did you mean to delete this file?

Reply:
Yep we moved the homebrew stuff to `scripts/generate-homebrew-formula.js`. Looks like that other file wasn't required anymore.